src/components/LoginPage/Slide4.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import slideImg1 from 'components/LoginPage/images/slide-4_img-1.svg';
+import slideImg2 from 'components/LoginPage/images/slide-4_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 10%;
+  text-align: center;
+  position: relative;
+
+<<<<<<< HEAD
+=======
+    .shadow {
+    -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    }
+>>>>>>> 0dfe1e73b0b826d4e2cba58197e330290b9fcca4
+    .img1 {
+        position: absolute;
+        width: 60%;
+        right: 0%;
+        top: 3%;
+        z-index: 1;
+    }
+    .img2 {
+        width: 100%;
+        margin-left: -24%;
+<<<<<<< HEAD
+        -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+        filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+=======
+>>>>>>> 0dfe1e73b0b826d4e2cba58197e330290b9fcca4
+    }
+`;
+const Figure = styled.figure`
+  position: relative;
+  width: 74%;
+  margin: auto;
+  height: auto;
+`;
+const Img = styled.img`
+  width: 100%;
+  height: auto;
+`;
+const Caption = styled.svg`
+  font-weight: ${props => props.fontWeight || 300};
+  position: absolute;
+  width: ${props => props.width};
+  top: ${props => props.top};
+  left: ${props => props.left};
+  right: ${props => props.right};
+  bottom: ${props => props.bottom};
+`;
+
+const CaptionSvgTextBox = ({ viewBoxWidth = 36, viewBoxHeight = 22, ...rest }) => {
```

Comment:
`CaptionSvgTextBox` is declared in every Slide file. Since it's a component that is shared between all the slides, might make sense to move it out to a separate file and import in every Slide file.

Reply:
I am agree with this. Will done. Only one question - where in project tree it would be better to place this separated file?

Reply:
`src/components/LoginPage/fileName.jsx`

Not sure how to name the file though :)

or maybe create a separate file for each component: `src/components/LoginPage/CaptionSvgTextBox.jsx`

Reply:
It seems to me that it is better not to split this component. It performs one function - allows you to insert resizable text. But if you think otherwise, I will gladly accomplish this task the way you see fit)

Reply:
I'm not proposing to split this component :)

Reply:
I took a closer look, `CaptionSvgTextBox` basically only sets the default `viewBoxWidth` and `viewBoxHeight` for the `Caption` component, so it's fine to declare it in every slide-file, because we use different defaults and are reusing the `Caption` component.

As long as we move the `Caption` component to a separate file, it should be fine. For `Caption` we're currently duplicating a lot of props in every slide-file.

Reply:
Done in commits [37c53ee](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/37c53ee2dddeca76a75a008408c6a511c2e6412c) & [47db387](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/47db387878fd98a78769a9cc27959a1f075f46c3)